lib/logitech_receiver/base.py

```diff
@@ -267,11 +272,23 @@ def _skip_incoming(handle, ihandle, notifications_hook):
 def make_notification(devnumber, data):
 	"""Guess if this is a notification (and not just a request reply), and
 	return a Notification tuple if it is."""
+
 	sub_id = ord(data[:1])
 	if sub_id & 0x80 == 0x80:
 		# this is either a HID++1.0 register r/w, or an error reply
 		return
 
+	# regular keyboard key press reports and mouse movement reports are not notifications
+	# it would be better to check for report_id 0x20 but that information is not sent here
+	if ( len(data) == _MEDIUM_MESSAGE_SIZE-2  and  ( sub_id==0x02 or sub_id==0x01 ) ) :
```

Comment:
According to the spec:
- 0x00 - 0x3f are RF reports
- 0x40 - 0x7f are DJ notifications
- 0x80 - 0xff are DJ commands

So I think you can basically ignore the whole first range right?

Reply:
@Lekensteyn Probably.  I was worried that there are some RF reports that should be looked at - like the HID++ 1.0 battery reports.  However, if you think that there is no reason to ever create notifications for these I'm willing to be brave here.

Reply:
Done